File: src/targets/javascript/jquery/fixtures/query.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value',
=======
  url: 'http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value',
>>>>>>> upstream/master
  method: 'GET',
  headers: {}
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/csharp/httpclient/fixtures/query.cs
using System.Net.Http.Headers;
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Get,
<<<<<<< HEAD
    RequestUri = new Uri("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value"),
=======
    RequestUri = new Uri("https://httpbin.org/anything"),
>>>>>>> upstream/master
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}